### DISCUSSION TITLE
How do I get started?

### DISCUSSION THREAD

**Discussion by vijaya-lakshmi-venkatraman** [CoreWCF/CoreWCF#354] [Q&A]
Sorry for this basic question.

I see from the Readme section that I should add the below link to my solution's nuget.config file.
Please can someone help me on what happens next? what should I do in order to make my WCF server code migrated after this step?

**Comment by mconnew** [comment]
You can actually get it from nuget.org now. That isn't needed unless you need a recent bug fix which hasn't been pushed to nuget yet. Take a look in the Samples folder, you will see some example services. There is a sample service from .NET Framework, and the same service with .NET Core. That should give you an idea how to port across. There's a good overview in [this blog post](https://corewcf.github.io/blog/2021/02/19/corewcf-ga-release) which should be helpful too.
